openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfile.java

```diff
@@ -93,13 +94,16 @@
     private static final String KEY_PREPARE_CH_WEIGHTINGS = "prepare.ch.weightings";
     private static final String KEY_PREPARE_LM_WEIGHTINGS = "prepare.lm.weightings";
     private static final String KEY_PREPARE_CORE_WEIGHTINGS = "prepare.core.weightings";
+    private static final String KEY_PREPARE_FASTISOCHRONE_WEIGHTINGS = "prepare.fastisochrone.weightings";
```

Comment:
I am not sure how this would work. The RoutingProfile handles all requests, even the isochrones ones. Would you say just take all the isochrone code from RoutingProfile, put it in a separate class (including all the keys) and then call this class from routingprofile?

Reply:
I didn't look into into this in detail, so it might be unfeasible without major refactoring. What I thought was that some abstraction and separation could overall increase the maintainability of the code. Is a subclass `IsochroneRoutingProfile` an option at all?

Reply:
Some separation would be good, yes. Though I do not see it without major refactoring of the whole RoutingProfile. It basically manages everything right now. The whole system of app.config and routingprofile seems very messy to me and I do not think that extracting a few fast isochrone parameters now makes much of a difference

Reply:
Marking as resolved because refactoring `RoutingProfile` is outside of the scope of this PR.